sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -32,7 +32,7 @@ protected TextAnalyticsClient()
 
         /// <summary>
         /// Initializes a new instance of the <see cref="TextAnalyticsClient"/>
-        /// class for the specified service instance.
+        /// class for the specified service instance in Azure Public Cloud.
```

Comment:
I don't think that we need to make this distinction here.   Given that the majority of use is in the public cloud, my concern is that this may cause more confusion to the 90% case than it would be helpful for the 10% case.   My vote would be to just demonstrate use in other clouds via samples.